lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -874,6 +913,14 @@ func loadAllStatesForIMSIs(networkID string, imsis []string) (map[string]state_t
 	return statesBySid, nil
 }
 
+func shouldLoadState(validIMSIs map[string]struct{}, imsi string) bool {
```

Comment:
Nit: define this function as an anonymous function within the caller's body (assign it to a variable within `loadAllStatesForIMSIs`). Because
- It's only relevant there
- We're only defining it for clarity's purpose
- It can use variables from the caller's scope

Ref: https://gobyexample.com/closures